### ISSUE TITLE
Slay the Spire does not work

### ISSUE THREAD

**Issue by czyba** [flathub/com.valvesoftware.Steam#121]
Trying to run the game Slay the Spire eventually runs into the following error:

```
>>> Adding process 11197 for game ID 646570
Exception in thread "LWJGL Application" java.lang.ExceptionInInitializerError
	at com.badlogic.gdx.backends.lwjgl.LwjglGraphics.setVSync(LwjglGraphics.java:558)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:124)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
	at org.lwjgl.opengl.LinuxDisplay.getAvailableDisplayModes(LinuxDisplay.java:954)
	at org.lwjgl.opengl.LinuxDisplay.init(LinuxDisplay.java:738)
	at org.lwjgl.opengl.Display.<clinit>(Display.java:138)
	... 2 more
```
According to [this thread](https://steamcommunity.com/app/646570/discussions/1/1692669912407841411/) the xrandr utility is necessary in order for the game to run. If this is true, my conjecture is that the image does not contain the program.

Flatpak versions tried: 0.10.4, 0.99.3

**Comment by nanonyme** [comment]
Sounds like a game issue to me

**Comment by czyba** [comment]
I would agree. If the code would use a library instead of depending on an optional program, than this should probably work. 

If you wish, I will close this issue and simply mark an entry in the wiki as does not work.

**Comment by nanonyme** [comment]
We can leave this open but I'd say you should probably try to ping devs if at all possible?

**Comment by TingPing** [comment]
`xrandr` is another tool that I don't think is unreasonable to bundle.

**Comment by nanonyme** [comment]
Yeah, I guess in this specific case it might be harmless if it's just used for setting vsync or other simple things like that. I don't trust it working identically for XWayland and native X11 in all cases

**Comment by nanonyme** [comment]
Please re-test with next update

**Comment by czyba** [comment]
@nanonyme Thank you. I can confirm that it now works like a charm.

**Comment by gvlasov** [comment]
I fixed this problem on my machine. I had `xrandr`, but the game still wouldn't start with this exact error.

It was caused by LibGDX incorrectly parsing output of `xrandr -q`. I had a mode named `1920x1080_60.00` that I manually created because Debian didn't recognize my display's modes out of the box. I renamed that mode to `1920x1080`, and the game started properly.

This is where I got the fix: https://github.com/LWJGL/lwjgl/issues/112#issuecomment-291598776
